plat/renesas/rcar/platform.mk

```diff
@@ -13,6 +13,12 @@ GENERATE_COT			:= 1
 BL2_AT_EL3			:= 1
 ENABLE_SVE_FOR_NS		:= 0
 
+CRASH_REPORTING		:=	1
+$(eval $(call add_define,CRASH_REPORTING))
```

Comment:
You don't need `` $(eval $(call add_define,CRASH_REPORTING))`` or `` $(eval $(call add_define,HANDLE_EA_EL3_FIRST))``. This platform makefile is included before bl31.mk and before the list of assert_boolean and add_define in the Makefile.